raiden-dapp/tests/unit/utils/stub.ts

```diff
@@ -1,6 +1,7 @@
 export function stub<T>(): T {
   const typeAssertion = <T>{};
   for (const prop in typeAssertion) {
+    // @ts-ignore
```

Comment:
Please, don't, if not needed:

Reply:
I wouldn't if it was not needed, without it the build is crashing

Reply:
Removing it. See the comment bellow :)